### DISCUSSION TITLE
[2024-06-10] Team GUI Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#10227] [Weekly Demo Day]
* Worked on release; #10077 #10125
* #10109 - bug discovered during release process
* #10076
* #10127 - also pushed to 1.1
* #10179
* Implemented node renaming using LS endpoint, but it does not work.

[Screencast from 2024-06-10 15-54-47.webm](https://github.com/enso-org/enso/assets/3919101/e87ded37-83c1-4abc-9451-0d33587b00dd)

[Screencast from 2024-06-10 15-56-27.webm](https://github.com/enso-org/enso/assets/3919101/df2519ec-d559-49b8-bc82-820788643491)

[Screencast from 2024-06-05 11-40-50.webm](https://github.com/enso-org/enso/assets/3919101/8dcb9099-5489-488c-86dc-560325e84f52)

**Comment by Frizi** [comment]
- Implemented new tooltips design as Vue component. #10172 

https://github.com/enso-org/enso/assets/919491/85f5fef7-c6df-471b-b544-b45f45f4f51e

- Moved node output port below the visualization, improved edge rendering near it. Fixed conflicting hover area shapes for the ports and visualizations resize handles. #10113

![image](https://github.com/enso-org/enso/assets/919491/8ae50912-1dfb-4389-b51a-44022b334b89)

- Improved the layout of dropdowns. Limited width and implemented text scrolling, better behaviour nearby port ends or at the edge of screen. #10198

https://github.com/enso-org/enso/assets/919491/148d117a-8f2c-4176-b0c8-c7d2390173c3

**Comment by kazcw** [comment]
- #10058

https://github.com/enso-org/enso/assets/1047859/3f873f3f-31b1-44bf-ae3e-2f467f2d546d

- #10215--applied cursor changes; fixed some bugs I found in click handling and hover cursors; introduced a solution to avoid hover/click inconsistencies.

**Comment by vitvakatu** [comment]
- https://github.com/enso-org/enso/pull/10194. As a plus, the order of copied nodes now matches the order of code lines.

https://github.com/enso-org/enso/assets/6566674/c0c5a524-ea6a-41f6-9fea-c94431211f33

- Working on https://github.com/enso-org/enso/issues/9970

**Comment by hubertp** [comment]
Relevant engine changes:
- ydoc-server is a separate module #10156 

Debugging performance issues with IGV - https://drive.google.com/file/d/11o2eHSy0Ptux0SWW9JpRiGQp5wtVZshh/view?usp=sharing
